Swagger/PlayFab/Profiles.swagger.json

```diff
@@ -467,10 +467,6 @@
           "description": "The language on this profile.",
           "type": "string"
         },
-        "LeaderboardMetadata": {
```

Comment:
I'm checking on whether we want to actually publish this change. It has the potential to break people.

Reply:
@AlonsoMondal We'll want to add a release note for this, which means figuring out how to add release notes. :)